client/packages/coldchain/src/Equipment/DetailView/AppBarButtons.tsx

```diff
@@ -84,6 +77,12 @@ export const AppBarButtonsComponent = () => {
     const date = new Date().toLocaleDateString();
     setIsPrinting(true);
     try {
+      if (!window.BrowserPrint) {
```

Comment:
Why would this be the case ? There is also a check for the below in the if statement, do we think that bundle won't be present ?